rdmo/projects/assets/js/interview/constants/management.js

```diff
@@ -0,0 +1,6 @@
+export const managementHelp = gettext(
+    'These links take you directly to the items in the admin interface. ' +
```

Comment:
is the naming of admin interface consistent? it is the (content) management interface right?